lib/ui/compositing.dart

```diff
@@ -27,10 +27,13 @@ class Scene extends NativeFieldWrapperClass2 {
     if (width <= 0 || height <= 0) {
       throw Exception('Invalid image dimensions.');
     }
-    return _futurize((_Callback<Image> callback) => _toImage(width, height, callback));
+    return _futurize((_Callback<Image> callback) => _toImage(width, height, (_Image image) {
```

Comment:
The documentation on the method `toImage` needs updating explaining that you get a handle to an image that needs to be disposed when you're done with it.

Reply:
Added some more details here.